README.md

```diff
@@ -21,6 +21,19 @@ Developer Documents:
 * [Copyright and License Notices](./NOTICES.md)
 * [License](./LICENSE)
 
+## Configuration
+
+### RateLimit
+Basic rate limiting can be applied by supplying a `RateLimit` object in the `appsettings.json` file. If no `RateLimit` object is supplied, rate limiting is not configured for the application. Rate limiting (when applied) will be set globally and apply to all application endpoints. 
```

Comment:
Two recommendations:

1. Add a blank line above this, so that there's always a line between the heading and the text.
2. Add hard line wrapping.

Just a few weeks ago I published a [Markdown Best Practices](https://techdocs.ed-fi.org/display/ETKB/Markdown+Best+Practices) document, but forgot to share it with folks. Includes some useful VS Code extensions.